rocon_web_remocon/www/index.js

```diff
@@ -8,11 +8,29 @@ var ros = new ROSLIB.Ros();
 var gListRoles = [];
 var gListInteractions = [];
 var gFinalUrl;
+var gFinalHash;
 
 var gUrl;
 var gCookieCount;
 
-var defaultUrl;
+var defaultUrl = 'ws://192.168.10.24:9090';
+
+
+//Remocon profile
+var gPublishers = {}
+var gRunningInteractions = [];
+var gRoconVersion = 'acdc'; //todo make rocon/version.js fot obtaining
+var gRemoconUUID = generateUUID().replace(/-/g,'');
+var gRemoconName = 'web_remocon_' + gRemoconUUID;
+var gRemoconRoconURI = 'rocon:/*/' + gRemoconName + '/*/trusty|chrome'
```

Comment:
should utilise [getBrowser](https://github.com/robotics-in-concert/rocon_web_gui/blob/indigo/rocon_web_remocon/www/index.js#L577-L586) to get correct rocon uri.